testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -536,27 +536,13 @@
   step %(I check "Test Base Channel" in the list)
 end
 
-When(/^I check the child channel "([^"]*)"$/) do |channel|
-  find(:xpath, "//i[@class='fa fa-angle-right']").click unless has_xpath?("//i[@class='fa fa-angle-down']", wait: 60)
-  checkbox = find(:xpath, "//label[contains(.,'#{channel}')]/..//input", match: :first, wait: 60)
-  checkbox.set(true)
-end
-
-When(/^I check the custom channels for "([^"]*)"$/) do |client|
-  node = get_target(client)
-  _os_version, os_family = get_os_version(node)
-  if os_family =~ /^ubuntu/
-    steps %(
-      When I check the child channel "main"
-      And I check the child channel "main-updates"
-    )
-  elsif os_family =~ /^centos/
-    step %(I check the child channel "DVD")
+When(/^I add "([^"]*)" to the child channels to migrate$/) do |channel|
+  within(:xpath, "//ul[@class='list-channel']/li[strong/text()='Optional Child Channels:']") do
```

Comment:
I would prefer something simpler in this xpath. First, if I'm not wrong there is only one place in the section where we can match a text '#{channel}' followed by an input element, so there is no need for this within statement. Second, it would be enough to filter using the class name 'list-channel' without including the text part, as this text has more probabilities to be changed.

Reply:
1) Good remark. Right, but I would remove the "list-channel" instead

2) As Can said, relying on internals is bad practice, it's better to rely on something visible by the user
  (and internals change too)

Reply:
I prefer internals than text for filtering like here, I don't consider a bad practice to rely on ids and classnames, anyhow as said, this is a completely unnecessary filter.

Reply:
yes it's unnecessary.

Reply:
done